pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -265,21 +265,31 @@ class Build {
 
                         def JobHelper = context.library(identifier: 'openjdk-jenkins-helper@master').JobHelper
 
+                        def testParameters = [
+                            context.string(name: 'UPSTREAM_JOB_NUMBER', value: "${env.BUILD_NUMBER}"),
+                            context.string(name: 'UPSTREAM_JOB_NAME', value: "${env.JOB_NAME}"),
+                            context.string(name: 'RELEASE_TAG', value: "${buildConfig.SCM_REF}"),
+                            context.string(name: 'JDK_REPO', value: jdkRepo),
+                            context.string(name: 'JDK_BRANCH', value: jdkBranch),
+                            context.string(name: 'OPENJ9_BRANCH', value: openj9Branch),
+                            context.string(name: 'LABEL_ADDITION', value: additionalTestLabel),
+                            context.string(name: 'KEEP_REPORTDIR', value: "${keep_test_reportdir}"),
+                            context.string(name: 'ACTIVE_NODE_TIMEOUT', value: "${buildConfig.ACTIVE_NODE_TIMEOUT}")]
+                        if (buildConfig.VARIANT == "openj9") {
+                            // PLATFORM only needs to be passed until all openj9 test platforms are mixed by default
+                            def arch = buildConfig.ARCHITECTURE
+                            if (arch == "x64") {
+                                arch = "x86-64"
+                            }
+                            def os = buildConfig.TARGET_OS
+                            testParameters.add(context.string(name: 'PLATFORM', value: "${arch}_${os}_mixed"))
```

Comment:
As per https://github.com/eclipse/openj9/pull/11951#issuecomment-782229036, `_mixed` must not be added for 32-bit platforms.

Reply:
Added an exception to the `if` above for 32. Trying to test it internally but Adopt should test it.